sql/hive/src/main/scala/org/apache/spark/sql/hive/hiveUDFs.scala

```diff
@@ -128,11 +129,10 @@ private[hive] class DeferredObjectAdapter(oi: ObjectInspector, dataType: DataTyp
   override def get(): AnyRef = wrapper(func()).asInstanceOf[AnyRef]
 }
 
-private[hive] case class HiveGenericUDF(
+case class HiveGenericUDF(
```

Comment:
is it possible to rewrite `HiveGenericUDF` with `Invoke`? Then we can simply use `RuntimeReplaceable`

Reply:
Seems quite complicated to handle the hive value mapping and function wrapping